content/en/agent/amazon_ecs/_index.md

```diff
@@ -88,9 +88,9 @@ Configure the task using either the [AWS CLI tools][12] or using the Amazon Web
 aws ecs register-task-definition --cli-input-json <path to datadog-agent-ecs.json>
 ```
 
-[1]: https://docs.datadoghq.com/resources/json/datadog-agent-ecs.json
-[2]: https://docs.datadoghq.com/resources/json/datadog-agent-ecs1.json
-[3]: https://docs.datadoghq.com/resources/json/datadog-agent-ecs-win.json
+[1]: /resources/json/datadog-agent-ecs.json
+[2]: /resources/json/datadog-agent-ecs1.json
+[3]: //resources/json/datadog-agent-ecs-win.json
```

Comment:
Using an extra slash here doesn't work for me. Was this intentional?

Reply:
nope, typo, good catch!